index.md

```diff
@@ -34,6 +34,10 @@ See our [detailed guide](install).
 * [Toolbox -- a fall 2019 update](https://debarshiray.wordpress.com/2019/11/01/toolbox-a-fall-2019-update/)
 * [Video talk -- Toolbox: using Silverblue for development](https://www.youtube.com/watch?v=BGXs0W6NRBM) by Debarshi Ray
 
+## Press Information
+We are very happy to answer questions from journalists and tech writers. The toolbox logo is licensed CCBYSA3.0 and can be [downloaded here](/assets/logo/toolbx-logo.zip).
+
+Press queries, including requests for comments and interviews can be directed to [press@containertoolbx.org](mailto:press@containertoolbx.org).
```

Comment:
Do we need to create a `press@containertoolbx.org` email alias or mailing list? How is it handled for the other projects?

The Podman folks might have something set up already. We could ask them to let us piggy back on their thing.

What do you think?

Reply:
Oh yea, we went with a placeholder live :) I was gonna have you replace it with what you got.

Reply:
We got `press@lists.podman.io` created and updated the website a few weeks back.